aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr.md

```diff
@@ -1,209 +1,182 @@
 ---
 uid: signalr/overview/getting-started/tutorial-getting-started-with-signalr
-title: "Tutorial: Getting Started with SignalR 2 | Microsoft Docs"
+title: "Real-time chat with SignalR 2 | Microsoft Docs"
 author: pfletcher
-description: "This tutorial shows how to use SignalR to create a real-time chat application. You will add SignalR to an empty ASP.NET web application and create an HTML pa..."
+description: "This tutorial shows how to use SignalR to create a real-time chat application. You will add SignalR to an empty ASP.NET web application and..."
```

Comment:
Something's missing here, I would just drop the and + ellipsis, and terminate the sentence with a period.